lib/segmented-controls.js

```diff
@@ -120,7 +120,7 @@ class SegmentedControls extends React.Component {
               borderBottomRightRadius: adjustedBorderRadius
             }
         }
-        if (index === this.props.options.length - 2) {
```

Comment:
Simply changing `if` to `else if` solves the problem.

Reply:
@eHammarstrom when this fix going to be merged? 
Thanks!